src/main/java/com/nerdtranslator/translateapibridge/service/impl/NerdTranslatorServiceImpl.java

```diff
@@ -49,6 +48,11 @@ public TranslationData getTranslation(String originalText, String originalLangua
                     List<String> partsOfSpeech = new ArrayList<>();
                     for (String word : translatedWords) {
                         String partOfSpeech = naturalLanguageApiService.analyzeText(word);
+                        /*todo
```

Comment:
Do you have a task for this in Trello? 
Let's move the description of the task to Trello and here will be only ToDo note with the link to this task